deploy/helm/sumologic/conf/events/events.conf

```diff
@@ -35,7 +35,7 @@
   @type sumologic
   @id sumologic.endpoint.events
   sumo_client {{ include "sumologic.sumo_client" . | quote }}
-  endpoint "#{ENV['SUMO_ENDPOINT_EVENTS']}"
+  endpoint "#{ENV['SUMO_ENDPOINT_EVENTS_SOURCE']}"
```

Comment:
Would this be considered a breaking change for non-helm users? Helm should handle this, but what abotu non-helm.

Reply:
actually since non-helm users just run `kubectl apply -f fluentd-sumologic.yaml` to upgrade, the new yaml file should have the renamed environment variables so it should be fine.

However there IS a breaking change here, and that is a result of the keys in the sumologic secret being changed (added suffix `-source`), which means that non-helm users will have to re-run the setup job in order to get an updated secret (though the collector and sources will remain untouched).

Reply:
Good catch, yeah we dont what the breaking change from the secret.